packages/babel-helper-wrap-function/src/index.ts

```diff
@@ -6,19 +6,39 @@ import {
   callExpression,
   functionExpression,
   isAssignmentPattern,
+  isFunctionDeclaration,
   isRestElement,
   returnStatement,
 } from "@babel/types";
 import type * as t from "@babel/types";
 
+type ExpressionWrapperBuilder<ExtraBody extends t.Node[]> = (
```

Comment:
I have never work on a project where we needed such a custom type. It leave me suspicious.